frameworks/hdfs/tests/test_upgrade.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+import pytest
+import sdk_install
+import sdk_upgrade
+import sdk_utils
+
+from tests import config
+
+log = logging.getLogger(__name__)
+
+foldered_name = config.FOLDERED_SERVICE_NAME
+
+
+@pytest.mark.sanity
+def test_hdfs_upgrade():
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+
+        if sdk_utils.dcos_version_less_than("1.9"):
```

Comment:
you can remove this if block. We're only testing on 1.11 and newer these days.

Reply:
Done